modules/AlphaWalletFoundation/AlphaWalletFoundation/Types/SessionsProvider.swift

```diff
@@ -193,7 +193,18 @@ public class ApiTransporterFactory {
         if let transporter = transportes[server] {
             return transporter
         } else {
-            let transporter = BaseApiTransporter()
+            let policy: RetryPolicy
+
+            switch server {
+            case .goerli, .mumbai_testnet:
+                //NOTE: goerli as well as mumbai_testnet retrun 403 error code
+                policy = ApiTransporterRetryPolicy(retryableHTTPStatusCodes: [429, 408, 500, 502, 503, 504, 403])
```

Comment:
In practise, does it work if we retry for these 3 chains after a `403` — so it's really just an intermittent server error?

Reply:
no idea, but after adding retyring it started to work for me. i tried different ways, changed headers but still has same issue. the problem is that sending same request (i copied curl) from postman or terminal works fine.